### PR TITLE
Feature/mock util cmd

### PR DESCRIPTION
**Pull request by Bluebugs**
### Description:
This patch series introduce the ability to actually test the behavior of generating Package and use it to validate that packaging for web, gopherjs and wasm work and produce the same things over time.

### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: cmd/fyne/internal/commands/get.go [hunk 1]
```diff
@@ -8,7 +8,6 @@ import (
 	"path/filepath"
 	"strings"
 
-	"fyne.io/fyne/v2/cmd/fyne/internal/util"
```
Comment (andydotxyz) [inline cmd/fyne/internal/commands/get.go:11]: This (plus others later in) cannot be right?
Files don't compile if the import isn't used - so either this didn't compile before (actions should have caught it) or it doesn't compile now...

Comment (andydotxyz) [reply cmd/fyne/internal/commands/get.go:11]: I'm not sure how it's possible that the actions were passing before and after this change - can you explore?

**Review by andydotxyz — CHANGES_REQUESTED** [review]

File: cmd/fyne/internal/commands/get.go (hunk 1 above)
Comment (Bluebugs) [reply cmd/fyne/internal/commands/get.go:11]: They are right. That's how mock work, by intercepting the access to io operation and allowing them to get verified on how they are called and what they return. The package util file will do the default redirection, when no mock are in place while the package util mock test will do that redirection when running the tests.

Comment (Bluebugs) [reply cmd/fyne/internal/commands/get.go:11]: And yes, it does compile :-)

Comment (andydotxyz) [reply cmd/fyne/internal/commands/get.go:11]: My apologies, I completely missed that is how it works. Very slick :)

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**